src/Main.java

```diff
@@ -1,5 +1,45 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+/*
+Создать программу для чтения номеров контрактов из файла.
+Неповторяющиеся номера контрактов следует сохранить в структуру
+данных ключ-значение, где ключ - это номер контракта, а значение -
+это объекты класса Document со всей контекстной информацией
+(номер документа, дата создания и другая информация).
+Структуру данных ключ-значение вывести на экран. Использовать коллекции
+Set и Map.
+* */
+
 public class Main {
-    public static void main(String[] args) {
-        System.out.println("Hello world!");
+    public static void main(String[] args) throws IOException {
+        Map<String, Document> map = new HashMap<>();
+        Set<String> contractSet = new HashSet<>();
+        try (BufferedReader br = new BufferedReader(new FileReader("Document.txt"))) {
+            String array;
```

Comment:
Minor, but veri important note:
your variables should have names based on the information, which you put inside.
What I mean here -> `array` means that you are going to store an array of elements, but in your case it's not an array!